arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -12,65 +12,117 @@ This how-to provides step-by-step instructions for running an Orbit node on your
 
 - **RAM**: 8-16 GB
 - **CPU**: 2-4 core CPU (For AWS: t3 xLarge)
-- **Storage**: It depends on the Orbit chain and its traffic during the time.
+- **Storage**: This depends on the Orbit chain and its traffic overtime.
 
 ### Prerequisites
 
 - Latest Docker Image: <code>@latestNitroNodeImage@</code>
 
-### Required parameters
+## Required parameters
 
-- Parent chain RPC URL
-  - Use the parameter `--parent-chain.connection.url=<Parent chain RPC URL>`
-  - It must provide a standard RPC endpoint for an EVM node, whether self-hosted or obtained from a node service provider
-    - Would be set to appropriate Arbitrum One, Arbitrum Nova, or Arbitrum Sepolia RPC URL
-  - Note: [Arbitrum RPC endpoints](/node-running/node-providers.mdx/#rpc-endpoints) rate limit connections, So you will need to run a local node for the parent chain or rely on third-party RPC providers
-- L3 Orbit chain info and chain name
-  - Providing the flag --chain.info-json=<Orbit Chain's chain info> is mandatory. The chain info json string contains vital information about the Orbit chain, which is necessary for the node to commence operations. An example of the chain info json string is available in the subsequent sections.
-  - `--chain.name=<My Arbitrum L3 Chain>` must be provided, and it must match the chain name that is used in `--chain.info-json` content.
-  - For an Anytrust chain, it is essential to provide `--node.data-availability.enable`.
+### 1. Parent chain parameters
 
-### Important ports
+The `parent-chain` argument needs to provide a standard RPC endpoint for an EVM node, whether self-hosted or obtained from a node service provider:
+
+```bash
+--parent-chain.connection.url=<Parent chain RPC URL>
+```
+
+The RPC endpoint must direct to <a data-quicklook-from="arbitrum-one">Arbitrum One</a>, <a data-quicklook-from="arbitrum-nova">Arbitrum Nova</a>, or<a data-quicklook-from="arbitrum-sepolia">Arbitrum Sepolia</a>.
+
+:::note
+
+[Arbitrum RPC endpoints](/node-running/node-providers.mdx/#rpc-endpoints) rate-limit connections. To avoid hitting a bottleneck, you can  run a local node for the parent chain or rely on third-party RPC providers.
+
+::: 
+
+### 2. Child chain parameters 
+
+
+In the Arbitrum Orbit context, the child chain is an L3 Orbit chain, and the required parameters are `chain-info` and `chain-name`
```

Comment:
This is not always the case. An Orbit chain can be an L2.

Reply:
Merged.